py/autoevals/moderation.py

```diff
@@ -0,0 +1,94 @@
+import threading
+
+from braintrust_core.score import Score, Scorer
+
+from .oai import arun_cached_request, run_cached_request
+
+REQUEST_TYPE = "moderation"
+
+
+class Moderation(Scorer):
+    """
+    A scorer that uses OpenAI's moderation API to determine if AI response contains ANY flagged content.
+    """
+
+    _CACHE = {}
+    _CACHE_LOCK = threading.Lock()
+
+    threshold = None
+    extra_args = {}
+
+    def __init__(self, threshold=None, api_key=None, base_url=None):
+        """
+        Create a new Moderation scorer.
+
+        :param threshold: A prefix to prepend to the prompt. This is useful for specifying the domain of the inputs.
```

Comment:
I think this comment needs to be updated

Reply:
Fixed: https://github.com/braintrustdata/autoevals/pull/63/commits/4baacb6e39a8f05b09f8393b73ab082205e049c3#diff-059564dc34fe7366598e59c4b9fbcab770ba4689f9c9418010178055baed560eR20